neo/NeoSystem.cs

```diff
@@ -50,13 +63,18 @@ public void StartConsensus(Wallet wallet)
         public void StartNode(int port = 0, int wsPort = 0, int minDesiredConnections = Peer.DefaultMinDesiredConnections,
             int maxConnections = Peer.DefaultMaxConnections)
         {
-            LocalNode.Tell(new Peer.Start
+            start_message = new Peer.Start
             {
                 Port = port,
                 WsPort = wsPort,
                 MinDesiredConnections = minDesiredConnections,
                 MaxConnections = maxConnections
-            });
+            };
+            if (!suspend)
```

Comment:
While, this works, would it possibly be better to just have a way to check if the plugins were finished loading? Then the caller could just wait to call `StartNode` until they were finished instead of needing these functions to inhibit `StartNode` doing what it is named to do.

Reply:
Or another possibility is perhaps StartNode blocks until plugins finish loading and we could provide StartNodeAsync if running from an async context.